packages/app-mobile/jest.config.js

```diff
@@ -1,17 +1,9 @@
-// Test configuration
-// See https://jestjs.io/docs/configuration#testenvironment-string
+module.exports = {
+	preset: 'react-native',
 
-const config = {
-	preset: 'ts-jest',
+	testMatch: ['**/*.test.js'],
```

Comment:
> Wait, actually @personalizedrefrigerator set it up with ts-jest, is that right? In that case, yes please use this for this particular package.

I saw this a bit late. I'll fix that.

Reply:
I'm currently setting up app-mobile to use ts-jest for testing and I've realized that the preset doesn't need to be 'ts-jest' for ts-jest to be used. The preset needs to 'react-native'. Just leaving this here to avoid any confusion. I followed this guide: [Using Typescript With React Native](https://reactnative.dev/blog/2018/05/07/using-typescript-with-react-native#adding-typescript-testing-infrastructure).

Reply:
I don't know about this, but I think that's going to conflict with what @personalizedrefrigerator is doing? @personalizedrefrigerator will your tests still work with this change?

Reply:
Currently, `tsconfig.json` is configured to not create .js files from `.test.ts` files. As such, unless `tsconfig.json` is modified, it shouldn't work with my test files.

Reply:
I changed `tsconfig` in my recent [commit](https://github.com/laurent22/joplin/pull/6662/commits/45962c7a4b982cec093fb81f1a3486adcfa10bbb#diff-dcc2d9d32ba5701b440eec8abf05054bb74f35ad002fa7c2e1107e964858f8d0). Your tests are running fine now as well as mine.

Reply:
I would like some consistency in the tests. Either it's all .js or it's all .ts but not mix and match. I liked the approach of running the .test.ts files directly because it means we don't need source maps.

Is it not possible to get that working with your tests @Tolu-Mals? If there's an issue please explain clearly what it is.

Reply:
It's possible. I'll do that.